pkg/utils/cluster.go

```diff
@@ -161,3 +161,28 @@ func AnnotateNode(ctx context.Context, nodeName string, key, value string, c cli
 
 	return AnnotateObject(ctx, node, key, value, c)
 }
+
+func AddLabelToNamespace(ctx context.Context, namespaceName, key, value string, c client.Client) error {
+	ns := &corev1.Namespace{}
```

Comment:
I was not able to find where we use this one.

in general I think we should document the need to add a label for monitoring on namespace creation and not add a rbac to allow the operator to update namespace object

Reply:
I removed the function. I will re-add it in the e2e test PR. 

About permissions, the operator already had the RBAC to write on namespaces (see [deploy/clusterrole.yaml](https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/108ef15a740dc226a6e217d6dedc9b3008ef8bb1/deploy/clusterrole.yaml#L19) and [openshift CSV](https://github.com/openshift/sriov-network-operator/blob/master/manifests/stable/sriov-network-operator.clusterserviceversion.yaml#L301)). No permission has been added in this PR for namespaces.

I'll take care of documenting the namespace configuration in OpenShift